Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -23,10 +23,10 @@
 from Orange.preprocess import RemoveNaNClasses
 from Orange.widgets import widget, gui, settings
 from Orange.widgets.utils.itemmodels import DomainModel
-from Orange.widgets.widget import OWWidget, Msg
+from Orange.widgets.widget import OWWidget, Msg, Input, Output
 
-Input = namedtuple(
-    "Input",
+InputLearner = namedtuple(
```

Comment:
Unrelated to the change --- needed just to avoid a name collision with the new class `Input`.